gradle.properties

```diff
@@ -4,12 +4,11 @@ VERSION_NAME=1.0.0-SNAPSHOT
 LATEST_VERSION=0.13.2
 # Versions
 ANIMALS_SNIFFER_VERSION=1.5.0
-ARROW_VERSION=0.13.2
+ARROW_VERSION=1.0.0
```

Comment:
@nomisRev i've bumped this from 0.13.2 to 1.0.0 - if i remember correctly they both should be backward compatible

Reply:
i think i hit another problem with the ank-gradle-plugin with 1.0.1 so i'm keeping it 1.0.0 for the time being, let's address it in the new issue